index.html

```diff
@@ -175,6 +175,21 @@ <h3 class="pre-heading__text">Conference venue</h3>
 </div>
 <!-- End Venue Section-->
 
+<div class="venue tb_padding">
+  <div class="container">
+    <div class="venue-2021-img venue-img">
+      <img src="/images/baja-roma.jpg">
+    </div>
+    <div class="location" class="row-item">
+      <h3 class="pre-heading__text">Closing Dinner</h3>
+      <p class="content-text">This year's Closing Dinner will be held at <a href="http://www.bajaroma.it/">Baja Roma</a> Friday night. After it has become an EmberFest tradition in the past years, we're doing on a boat again! The location is semi-outdoors with opening walls to ensure good airflow – all other <a href="/corona-safety/" style="display: inline;" class="content-link">Corona-related safety measures</a> apply as well of course.</p>
+      <div class="map-block">
+        <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2969.1332163885168!2d12.46982365056218!3d41.911494871201555!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x132f60f7627b6457%3A0xf04965480f9cb18!2sBaja!5e0!3m2!1sde!2sde!4v1631175601156!5m2!1sde!2sde" width="100%" height="100%" style="border:0;" allowfullscreen="" loading="lazy"></iframe>
```

Comment:
The map has German labels, could it be because of the `de` at the end?

Reply:
Hm, don't know – can't choose locale anywhere for the embed code

Reply:
I just tried removing those `de`s from the URL (only the trailing one wasn’t enough), and it works:

```suggestion
        <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2969.1332163885168!2d12.46982365056218!3d41.911494871201555!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x132f60f7627b6457%3A0xf04965480f9cb18!2sBaja!5e0!3m2!1s!2s!4v1631175601156!5m2!1s!2s" width="100%" height="100%" style="border:0;" allowfullscreen="" loading="lazy"></iframe>
```

![image](https://user-images.githubusercontent.com/1042339/132693744-dab9f6ce-95b9-4638-87e8-62b35b7ba74e.png)